packages/discovery-react-components/src/index.tsx

```diff
@@ -14,3 +14,4 @@ export {
 } from './components/DocumentPreview/components/HtmlView/HtmlView';
 export { CIDocument, canRenderCIDocument } from './components/CIDocument/CIDocument';
 export { default as StructuredQuery } from './components/StructuredQuery/StructuredQuery';
+export { getDocumentTitle } from './components/SearchResults/utils';
```

Comment:
We need to think about how we're exporting things from components, since these are our API. We have the top level components, but utility methods like this one should probably be namespaced somehow. Let's discuss with rest of team tomorrow.